Automattic/wp-calypso
# Gutenberg: replace copied core-blocks with npm package

Resolves: https://github.com/Automattic/wp-calypso/issues/26622

This replaces the copied `core-blocks` folder with recently published `@wordpress/block-library` npm package. 

### Testing instructions
1. Navigate to `/gutenberg/post/{siteSlug}`.
2. Verify that you can add a new block.
